htdocs/web_portal/css/web_portal.css

```diff
@@ -40,15 +38,21 @@ background-color: #8dbdd8;
 
 
 
-body {
-	background-image: url('../img/background3.png');
-	background-repeat: no-repeat;
-	background-color: #d0dff9;
+body {  
+	background: linear-gradient(to bottom left,#F7F9FE, #DEE9FB, #D0DFF9);
 	color: #272A4B;
-	font-family: "VAGRoundedLtRegular",Tahoma, "museo-sans-1", "museo-sans-2", "Helvetica Neue", Geneva,sans-serif;
-	font-size: 1.3em;
+	font-family: 'PT Sans', sans-serif;
```

Comment:
To use this font you will need to add:
```html
<link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
to the `<head>` section of the main renderation code-a-majig.

Reply:
Specifically edit htdocs\web_portal\static_php\standard_header.php, and add that line at the bottom of similar lines. You should then see the font change.

Reply:
Now fixed